dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -0,0 +1,174 @@
+import { faker } from "@faker-js/faker";
+import { LksgData, ProductionSite } from "../../../../build/clients/backend";
+
+import { randomYesNoUndefined } from "../common/YesNoFixtures";
+
+export function generateProductionSite(): ProductionSite {
+  const fakeSiteName = faker.company.name();
+  const yesNo = randomYesNoUndefined();
+  const fullFormattedFakeAddress =
+    faker.address.street() +
+    " " +
+    faker.address.buildingNumber() +
+    ", " +
+    faker.address.zipCode() +
+    " " +
+    faker.address.city() +
+    ", " +
+    faker.address.country();
+  const fakeGoodsAndServices = Array.from(
+    { length: faker.datatype.number({ min: 1, max: 5 }) },
```

Comment:
I would also include 0

Reply:
Repaired.

Reply:
@SiegfriedSobkowiak but how can there be a plant, which offers 0 goods and services?
Or have I misunderstood the meaning of this field in the data dictionary?

Reply:
I agree there should usually be at least one thing that is being produced but I would include the 0 to test unexpected input.